pkg/crypto/keys/publickey_test.go

```diff
@@ -179,3 +182,37 @@ func TestUnmarshallJSONBadFormat(t *testing.T) {
 	err := json.Unmarshal([]byte(str), actual)
 	require.Error(t, err)
 }
+
+func TestRecoverSecp256r1Static(t *testing.T) {
+	//these data were taken from the reference testcase
+	b := []byte{123, 245, 126, 56, 3, 123, 197, 199, 26, 31, 212, 186, 120, 195, 168, 153, 57, 108, 234, 49, 107, 203, 44, 207, 185, 212, 187, 129, 74, 43, 225, 69}
```

Comment:
Link?

Reply:
Yep, a reference would be helpful.

Reply:
I added link and wrote a short comment how we can update these values.